docs/atlascli/command/atlas-clusters-upgrade.txt

```diff
@@ -0,0 +1,103 @@
+.. _atlas-clusters-upgrade:
+
+======================
+atlas clusters upgrade
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Upgrade a MongoDB Atlas cluster.
+
+Syntax
+------
+
+.. code-block::
+
+   atlas clusters upgrade [clusterName] [options]
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterName
+     - string
+     - false
+     - Name of the cluster to upgrade
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --diskSizeGB
+     - float
+     - false
+     - Capacity, in gigabytes, of the host's root volume.
+   * - -f, --file
+     - string
+     - false
+     - File name to use, optional file with a json cluster configuration.
+   * - -h, --help
+     - 
+     - false
+     - help for upgrade
+   * - --mdbVersion
+     - string
+     - false
+     - MongoDB version of the cluster to deploy.
```

Comment:
We should take out the "to deploy" since this no longer on applies to deploying new ones
```suggestion
     - MongoDB version of the cluster.
```